helloworld-jms/README.adoc

```diff
@@ -117,9 +102,9 @@ timestamp org.jboss.as.quickstarts.jms.HelloWorldJMSClient main
 INFO: Received message with content Hello, World!
 ----
 
-== Optional Properties
+=== Optional Properties
```

Comment:
Personally I would get ridden of those (here and in the pom.xml) and hardcode them in the test class, like other QS are doing. The only non constant would be the java.naming.provider.url but you could derivate the value for that one from the server.host param used in tests

Reply:
I put them here for retro compatibility